### ISSUE TITLE
User avatars of other persons are not being displayed.

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#632]
Accepting invitation form needs to be updated in look and feel and capture person's name.

Problem, in completing the account settings, we get the following on screen error because user's name is blank in database because name not captured in sign-up from invite. 

`There was an error executing the template 'account_settings' output document ''Your Settings' (at index 0)': 'NoneType' object has no attribute 'name'`

**Comment by gregelin** [comment]
Upon further review, the larger issue is that we are trying to use GovReady-Q questionnaire to create the user profiles. That add complexity for accessing information about the users. We need to switch to a traditional user model.

**Comment by gregelin** [comment]
Temporarily fixed the display of other users' avatar thumbnails by adding following to `def task_view(view_func)` to allow all users to access avatar thumbnails.

```
# Yes in the special case of user accessing the avatar of another user
# TODO: Re-evaluate this code block after implementing a more traditional
#       user profile model instead of using a Q module for a persons profile.
#       see https://github.com/GovReady/govready-q/issues/632
if question_key == "picture":
   return True
```

**Comment by gregelin** [comment]
See #633  Upon further review, the larger issue is that we are trying to use GovReady-Q questionnaire to create the user profiles. That add complexity for accessing information about the users. We need to switch to a traditional user model.

**Comment by gregelin** [comment]
Code block to fix issue is in commit 68ceec39c5d82ab17a66bd494735890e53c3559e

**Comment by gregelin** [comment]
Fix of this issue will be in release of 0.9.0
